src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_decorator.py

```diff
@@ -0,0 +1,5351 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
What's this file for?

Reply:
removed. accidentally got there when resolve merge conflict